### PR TITLE
Move unknown app icon to other assets

### PR DESCRIPTION
**Pull request by meodai**

### CODE REVIEW

File: src/frontend/src/flows/verifiableCredentials/allowCredentials.ts [hunk 1]
```diff
@@ -26,9 +26,10 @@ const getOrigin = (origin: string, dapplist: KnownDapp[]): KnownDapp => {
     foundDapp = new KnownDapp({
       name: origin,
       website: origin,
-      logo: "unwnowndapp.png",
+      logo: "../unwnowndapp.png",
```
Comment (frederikrothenberger) [inline src/frontend/src/flows/verifiableCredentials/allowCredentials.ts:29]: Could we take this opportunity to rename the file to `unknowndapp.png`?

Comment (peterpeterparker) [inline src/frontend/src/flows/verifiableCredentials/allowCredentials.ts:29]: Randomly noticed this PR, can't you import the image given it finds place in src folder?

e.g. with vite / SvelteKit

```
import eth from '$lib/assets/eth.svg';

<img src={eth} />
```

Comment (meodai) [reply src/frontend/src/flows/verifiableCredentials/allowCredentials.ts]: @peterpeterparker I got a bitmap, but I will get an SVG soon from Artem and change it them. Hope that works.

**Review by frederikrothenberger — APPROVED** [review]
Thanks!

File: src/frontend/src/flows/verifiableCredentials/allowCredentials.ts (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/flows/verifiableCredentials/allowCredentials.ts]: I don't mean the extension, I mean using an `import`. If you use an import, your editor will be able to resolve it for you if you ever moved again the file in the future. Furthermore, it's then easier to find where the image is used.

But not sure it is supported in II.

**Pull request merged**